MMM-GoogleCalendar.js

```diff
@@ -1041,6 +1041,17 @@ Module.register("MMM-GoogleCalendar", {
         event.calendarName = this.calendarNameForCalendar(calendarID);
         event.color = this.colorForCalendar(calendarID);
         delete event.calendarID;
+
+        // Make a broadcasting event to be compatible with the default calendar module.
+        event.title = event.summary;
+        event.fullDayEvent = (event.start?.date && event.end?.date) ? true : false;
+        if (event.fullDayEvent) {
+          event.startDate = moment(event.start?.date).valueOf();
+          event.endDate = moment(event.end?.date).valueOf();
+        } else {
+          event.startDate = moment(event.start?.dateTime).valueOf();
+          event.endDate = moment(event.end?.dateTime).valueOf();
```

Comment:
I've noticed a potential edge case in the handling of `event.start?.dateTime` and `event.end?.dateTime` that may need additional consideration.

Should either event.start?.dateTime or event.end?.dateTime be `null` or `undefined`, moment(null/undefined) will return the current date and time (according to Moment.js documentation). This may not be the desired behavior in the context of our application.

It could be beneficial to define what should happen in these cases. Perhaps `event.startDate` and `event.endDate` should remain `null` or `undefined` to accurately reflect that no start or end time is available. Example:

```suggestion
        } else {
          event.startDate = event.start?.dateTime ? moment(event.start.dateTime).valueOf() : null;
          event.endDate = event.end?.dateTime ? moment(event.end.dateTime).valueOf() : null;
```

Reply:
I think it is probably checked before using `moment` by 
```js
event.fullDayEvent = (event.start?.date && event.end?.date) ? true : false;
```

Anyway, there is a chance the data is corrupted. I'll add more logics.